sites/feedback/page-feedback-contact-fr.html

```diff
@@ -0,0 +1,27 @@
+---
+{
+	"altLangPage": "page-feedback-contact-en.html",
+	"breadcrumbs": [
+		{ "title": "Zone de rétroaction - Documentation", "link": "sites/feedback/feedback-docs-fr.html" }
+	],
+	"dateModified": "2023-04-12",
+	"description": "Outil de rétroaction sur la page pour Canada.ca",
+	"language": "fr",
+	"title": "Outil de rétroaction sur la page avec lien de contact",
+	"share": "true",
+	"feedbackContact": {
+		"link": "Contacter Nom du service",
+		"url": "https://canada.ca/fr"
+	},
+	"feedbackData": {
+		"theme": "Thème",
+		"section": "Section"
+	}
+}
+---
+<div class="wb-prettify all-pre hide"></div>
```

Comment:
Considering we are going in provisional mode that are soft launch, let add the 2 alert on our page

```suggestion
{% include alert-provisional.html siteroot="../../" %}
{% include alert-softlaunch.html genre="e" component="page feedback tool" version="provisional" %}
<div class="wb-prettify all-pre hide"></div>
```